src/storage/src/hummock/utils.rs

```diff
@@ -559,6 +559,9 @@ mod tests {
 
     use crate::hummock::utils::MemoryLimiter;
 
+    // This is a clippy bug, see https://github.com/rust-lang/rust-clippy/issues/11380.
+    // TODO: remove `allow` here after the issued is closed.
+    #[allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
```suggestion
    #[expect(clippy::needless_pass_by_ref_mut)]
```

so that we can be notified after its getting fixed.